src/components/layout/index.tsx

```diff
@@ -22,12 +22,12 @@ export function Layout() {
         })} flex h-full w-full overflow-hidden py-[14px] px-[14px]`}
       >
         <SideNav />
-        <div className="px-[30px] w-full h-auto overflow-y-auto flex flex-col">
```

Comment:
why overflow-y-auto removed?

Reply:
As a result of utilizing 'overflow-y-auto', an unintended vertical scrolling issue is occurring specifically on Mac screens.